### ISSUE TITLE
USTP: extra blank image containers in FM

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#2186]
After yesterday's mammoth NEF-fest I have now downloaded the workstations. There are a ton of extra blank image containers created by sys with yesterday's date (I prepared this morning). Looks like it might be one for each file added from the photo import, and they are in between the iPad created trench photos and the photo import ones. Could be an artifact of the NEFs? I will do a round with jpegs today and see if the same thing happens, but it's a bug either way.
![image](https://github.com/arch-kiosk/arch-kiosk-office/assets/49291909/08a06727-b916-4d5c-8188-af9f4fd234d8)

**Comment by luizaogs** [comment]
Ja. I will do this in a bit, after finishing my application to graduate (scary).

**Comment by urapadmin** [comment]
okay, thanks. That cannot be right.

**Comment by lbestock** [comment]
Well you have found out what it was, and so my addition is useless but I will make it anyway! I did a mammoth JPG import. No problem. I did another (smaller) NEF import. No problem. I went and deleted a NEF from the File Repository and voila, next download had the empty image. So additional confirmation that @urapadmin you were right in the kitchen that it was your deletion yesterday rather than the NEFiness of it all that led to that.

**Comment by urapadmin** [comment]
okay, I looked this up. This is indeed what we are doing. The images containers are emptied (so the reference to the image that's about to be dropped is removed) but we do not remove the record itself. The rational behind this must have been that I did not want to lose potential data in the same record. And I can think of at least one example where that is correct: locus relations. The whole relation would be deleted if the image gets deleted. 

So this works as designed right now. Whether we want to improve it and how that can be achieved is something I postpone...

**Comment by urapadmin** [comment]
this was postponed and would still be the case: Deleting images does not delete the recording records and hence leads to empty containers in the records of the recording app. That is technically correct and in many cases what we want. But after a bulk import that went wrong it leaves a lot of fragments. Needs an idea how to deal with that.
